lldb/tools/lldb-vscode/README.md

```diff
@@ -212,6 +212,38 @@ This loads the coredump file `/cores/123.core` associated with the program
 }
 ```
 
+### Connect to a Debug Server on the Current Machine
+
+This connects to a debug server (e.g. `lldb-server`, `gdbserver`) on
+the current machine, that is debugging the program `/tmp/a.out` and listening
+locally on port `2345`.
+
+```javascript
+{
+  "name": "Local Debug Server",
+  "type": "lldb-vscode",
+  "request": "launch",
```

Comment:
Out of curiosity. Does launch actually work? Or does it need to be attach? IIUC you can only do launch if the server is running in platform mode, otherwise you can only attach.

Reply:
It does but I think because launch overriding launchCommands is equivalent to attach overriding attachCommands.

The only difference I see in VS Code is that in the start stop bar thing, launch has stop then disconnect, attach has disconnect then stop. Strange.

It's not attach or launch but it's more attach, so I'll change this.

Reply:
We give the example of "attaching" to a core file so we're already stretching the meaning.

Reply:
Changed them to use "attach" instead.